src/index.ts

```diff
@@ -13,44 +13,6 @@ export interface Event {
 export type Callback = (error: any, response: Twilio.Response) => void;
 export type HandlerFn = (context: Context, event: Event, callback: Callback) => void;
 
-/**
- * A validator to be used with Twilio Function. It uses the {@link validator} to validate the token
- *
- * @param handlerFn    the Twilio Runtime Handler Function
- */
-export const functionValidator = (handlerFn: HandlerFn): HandlerFn => {
```

Comment:
Moved this down to satisfy linting issue of using a function before it is defined